Formula/tezos-node-granadanet.rb

```diff
@@ -0,0 +1,89 @@
+# SPDX-FileCopyrightText: 2021 TQ Tezos <https://tqtezos.com/>
+#
+# SPDX-License-Identifier: LicenseRef-MIT-TQ
+
+class TezosNodeFlorencenet < Formula
```

Comment:
Looks like this should be changed

Reply:
right

Reply:
Fixed, but I accidentally squashed changes with the latest commit:(